### PR TITLE
BigQuery Common steps

### PR DESCRIPTION
**Pull request by neerajsinghal05**
BigQuery Common steps

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java [hunk 1]
```diff
@@ -1132,6 +1132,14 @@ public static void pressEscapeKey() {
     logger.info("Press Escape Key");
     Actions act = new Actions(SeleniumDriver.getDriver());
     act.sendKeys(new CharSequence[]{Keys.ESCAPE}).perform();
+  }
 
+  public static void enterKeyInDeDupe(String pluginProperty, String value) {
```
Comment (Aryan-Verma) [inline src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: nit:
```suggestion
  public static void enterKeyInDedupe(String pluginProperty, String value) {
```
Please change other instances as well

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java [hunk 2]
```diff
@@ -263,4 +263,16 @@ public static WebElement selectReplicationTable(String tableName) {
...
+    return SeleniumDriver.getDriver().findElement(By.xpath("//div[@data-cy='value']"));
+  }
+
+  public static WebElement locateTableKey(String pluginProperty) {
+    return SeleniumDriver.getDriver().findElement
+            (By.xpath("//*[@data-cy='" + pluginProperty + "']//input")); }
+
+  public static WebElement deDupeKey(String pluginProperty) {
+    return SeleniumDriver.getDriver().findElement(By.xpath
+            ("//*[@data-cy='" + pluginProperty + "']//input[@placeholder='Key']")); }
```
Comment (Aryan-Verma) [inline src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java:277]: Is Dedupe Key used anywhere else except BQ Sink plugin? If not, we can use the locator from  [CdfBigQueryPropertiesLocators](https://github.com/cdapio/cdap-e2e-tests/blob/5fced8e58371fff1539b8b9b30f9a5de4bfdcfb2/src/main/java/io/cdap/e2e/pages/locators/CdfBigQueryPropertiesLocators.java#L122) right?

File: src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java (hunk 1 above)
Comment (priyabhatnagar25) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Hi Aryan , I didnt get what is expected here.Can you please clarify here.

Comment (Aryan-Verma) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: It's just a small change in the casing: Dedupe instead of DeDupe

Comment (priyabhatnagar25) [reply src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java]: Resolved changed it to enterKeyInDedupe.

File: src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java (hunk 2 above)
Comment (priyabhatnagar25) [reply src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java:277]: So here we have changed the code for dedupe and relation table key, we are using both these values from pluginparameter file so , not using the already made dedupe and relation table key locator from CdfBigQuerProperties locators file.

**Review by Aryan-Verma — APPROVED** [review]

**Review by MrRahulSharma — APPROVED** [review]
LGTM

**Pull request merged**